be/src/exec/schema_scanner/schema_be_tablets_scanner.cpp

```diff
@@ -86,13 +97,24 @@ static const char* keys_type_to_string(KeysType type) {
     }
 }
 
+static const char* medium_type_to_string(TStorageMedium::type type) {
+    switch (type) {
+    case TStorageMedium::SSD:
+        return "SSD";
+    case TStorageMedium::HDD:
+        return "HDD";
+    default:
+        return "UNKNOWN";
+    }
+}
+
 Status SchemaBeTabletsScanner::fill_chunk(ChunkPtr* chunk) {
     const auto& slot_id_to_index_map = (*chunk)->get_slot_id_to_index_map();
     auto end = _cur_idx + 1;
     for (; _cur_idx < end; _cur_idx++) {
         auto& info = _infos[_cur_idx];
         for (const auto& [slot_id, index] : slot_id_to_index_map) {
-            if (slot_id < 1 || slot_id > 18) {
+            if (slot_id < 1 || slot_id > 19) {
                 return Status::InternalError(strings::Substitute("invalid slot id:$0", slot_id));
             }
             ColumnPtr column = (*chunk)->get_column_by_slot_id(slot_id);
```

Comment:
I'm curious that is there any way to optimize the code rather than hard coding? Such as using a map and a for loop.

Reply:
i have no idea